packages/js/product-editor/src/utils/register-product-editor-block-type.ts

```diff
@@ -20,7 +19,8 @@ export function useEvaluationContext( context: Record< string, unknown > ) {
 
 	const productId = useEntityId( 'postType', postType );
 
-	const getEvaluationContext = ( select: typeof WPSelect ) => {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	const getEvaluationContext = ( select: any ) => {
```

Comment:
How come we don't also declare the generic `SelectType` here so we don't need to use `any`?